Configuration/StandardSequences/python/Eras.py

```diff
@@ -60,7 +60,7 @@ def __init__(self):
         # Scenarios further afield.
         # Phase2 is everything for the 2023 (2026?) detector that works so far in this release.
         # include phase 1 stuff until phase 2 tracking is fully defined....
-        self.Phase2 = cms.ModifierChain( self.phase1Pixel, self.trackingPhase1, self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
+        self.Phase2 = cms.ModifierChain( self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
```

Comment:
how about

runTheMatrix.py -l 11024



> On Apr 26, 2016, at 3:41 PM, Lindsey Gray <notifications@github.com> wrote:
> 
> In Configuration/StandardSequences/python/Eras.py:
> 
> > @@ -60,7 +60,7 @@ def __init__(self):
> >          # Scenarios further afield.
> >          # Phase2 is everything for the 2023 (2026?) detector that works so far in this release.
> >          # include phase 1 stuff until phase 2 tracking is fully defined....
> > -        self.Phase2 = cms.ModifierChain( self.phase1Pixel, self.trackingPhase1, self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
> > +        self.Phase2 = cms.ModifierChain( self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
> 
> @slava77 Another way to fix this is to decommission this unit test and put HGCal tests into runTheMatrix or the addOn tests.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>